components/SettingsControls/src/Helpers/IsNullOrEmptyStateTrigger.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Specialized;
+
+#if WINAPPSDK
+using CommunityToolkit.WinUI.Helpers;
+#else
+using Microsoft.Toolkit.Uwp.Helpers;
+#endif
+
+
+namespace CommunityToolkit.Labs.WinUI;
+
+/// <summary>
+/// Enables a state if an Object is <c>null</c> or a String/IEnumerable is empty
+/// </summary>
+public class IsNullOrEmptyStateTrigger : StateTriggerBase
+{
+    /// <summary>
+    /// Gets or sets the value used to check for <c>null</c> or empty.
+    /// </summary>
+    public object Value
+    {
+        get { return GetValue(ValueProperty); }
+        set { SetValue(ValueProperty, value); }
+    }
+
+    /// <summary>
+    /// Identifies the <see cref="Value"/> DependencyProperty
+    /// </summary>
+    public static readonly DependencyProperty ValueProperty =
+        DependencyProperty.Register(nameof(Value), typeof(object), typeof(IsNullOrEmptyStateTrigger), new PropertyMetadata(null, OnValuePropertyChanged));
+
+    public IsNullOrEmptyStateTrigger()
+    {
+        UpdateTrigger();
```

Comment:
Out of curiosity (since we don't have the diff from the original), was this the only difference to fix the trigger?

Reply:
Yep! Tried adding the constructor first and add UpdateTrigger), but then saw that it defaulted to 'true' (bool). Changed this to 'null'and that did the trick.

As discussed, testing these kinds of uses cases would be good when we bring the triggers over :)